development/client/tsconfig.json

```diff
@@ -16,7 +16,8 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "noEmit": true,
-    "jsx": "react"
+    "jsx": "react",
+    "allowSyntheticDefaultImports": true
```

Comment:
Since `esModuleInterop` is already enabled, you shouldn't need `allowSyntheticDefaultImports`.

Reply:
Oh, it got taken out in the `testing` branch that got merged so I'll make sure it gets removed here too.